subprojects/resources/src/test/groovy/org/gradle/internal/resource/BuildOperationFiringExternalResourceDecoratorTest.groovy

```diff
@@ -141,7 +143,7 @@ class BuildOperationFiringExternalResourceDecoratorTest extends Specification {
         1 * buildOperationExecuter.call(_) >> { CallableBuildOperation op ->
             def operationContextMock = Mock(BuildOperationContext) {
                 1 * setResult(_) >> { ExternalResourceReadBuildOperationType.Result result ->
-                    assert result.bytesRead == TestExternalResource.READ_CONTENT_LENGTH
+                    assertTrue result.bytesRead == TestExternalResource.READ_CONTENT_LENGTH
```

Comment:
Why is this change needed?

Reply:
Same, a workaround to spockframework/spock#1232